### ISSUE TITLE
computing rest-frame magnitudes

### ISSUE THREAD

**Issue by themiyan** [cschreib/fastpp#5]
When i specify a filter in `REST_MAG` it still does not give the rest-frame magnitude as an output.

**Comment by cschreib** [comment]
Did you specify ```OUTPUT_COLUMNS``` manually? Or did you leave it empty (the default)? If you manually specified which columns to display, you naturally have to add the rest-frame magnitudes to this list for them to be displayed. As described in the documentation in the ```.param``` file, if you ask for the rest-frame magnitude in the filter with ID ```142```, then the associated column name is ```M142```.

Therefore if you had:
```
OUTPUT_COLUMNS = ['id','Av','lmass','sfr']
```

... you need to change it to:
```
OUTPUT_COLUMNS = ['id','Av','lmass','sfr','M142']
```

If this doesn't fix your problem then there's a bug, so please let me know if you still have issues!

**Comment by themiyan** [comment]
fixed as similar to #6 
`OUTPUT_COLUMNS = []` 
will also work.
